icu4j/main/core/src/main/java/com/ibm/icu/text/RuleBasedBreakIterator.java

```diff
@@ -168,7 +168,7 @@ public Object clone()  {
         RuleBasedBreakIterator result;
         result = (RuleBasedBreakIterator)super.clone();
         if (fText != null) {
-            result.fText = (CharacterIterator)(fText.clone());
+            result.fText = (CharacterIterator)fText.clone();
```

Comment:
do we need this cast?

Reply:
Yes, that is `java.text.CharacterIterator`, and the clone() returns an `Object`.
And I can't change that :-)